src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,180 @@
+"""
+Automation for default debug level
+
+:requirement: SSSD - Default debug level
+"""
+
+from __future__ import annotations
+
+import pytest
+from pytest_mh.ssh import SSHProcessError
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology, KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__check(client: Client):
+    """
+    :title: Check default debug level when sssd started successfully
+    :setup:
+        1. Clear logs and cache
+        2. Start SSSD with default debug level
+    :steps:
+        1. Check log files
+    :expectedresults:
+        1. "Starting with debug level = 0x0070" is in each file
+            Log message with number "0x3f7c0" is stored, if log contains more than one line
```

Comment:
This sentence is easier to understand if you reverse the order: `If log contains more than one line, log message with number "0x3f7c0" is stored`